### DISCUSSION TITLE
feat: How to Retrieve the Current Kairos Network Token?

### DISCUSSION THREAD

**Discussion by cheng990629** [kairos-io/kairos#2680] [Q&A]
Title:  How to Retrieve the Current Kairos Network Token?

Description:

I have lost my current Kairos network token and need to add a new node to the cluster. How can I retrieve the existing network token or generate a new one?

Thank you!

**Comment by mauromorales** [comment]
You should be able to access it via `kairos-agent config show` in one of the nodes
